lib/webdrivers/chromedriver.rb

```diff
@@ -23,13 +23,24 @@ def latest_version
         raise StandardError, 'Can not reach site' unless site_available?
 
         # Versions before 70 do not have a LATEST_RELEASE file
-        return Gem::Version.new('2.46') if release_version < '70.0.3538'
+        return normalize_version('2.46') if release_version < normalize_version('70.0.3538')
 
         # Latest webdriver release for installed Chrome version
-        release_file     = "LATEST_RELEASE_#{release_version}"
-        latest_available = get(URI.join(base_url, release_file))
-        Webdrivers.logger.debug "Latest version available: #{latest_available}"
-        @latest_version = Gem::Version.new(latest_available)
+        release_file = "LATEST_RELEASE_#{release_version}"
+        begin
+          latest_applicable = normalize_version(get(URI.join(base_url, release_file)))
+        rescue Net::HTTPServerException
+          latest_release = normalize_version(get(URI.join(base_url, 'LATEST_RELEASE')))
+          Webdrivers.logger.debug "Unable to find a driver for: #{release_version}"
+
+          msg = release_version > latest_release ? 'you are using a non-production version of Chrome; ' : ''
```

Comment:
Maybe the message should be 'You appear to be using a non-production version...'